packages/components/src/components/post-rating/post-rating.scss

```diff
@@ -0,0 +1,98 @@
+@use '@swisspost/design-system-styles/core' as post;
+
+:host {
+  display: inline-block;
+}
+
+.rating {
+  margin: 0 (-(post.$size-micro));
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  overflow: hidden;
+  & :focus {
+    outline: none;
+  }
+  &[aria-readonly='true'] .star-container {
+    cursor: default;
+  }
+}
+
+.rating:focus-visible {
+  outline-offset: post.$input-focus-outline-thickness;
+  outline: post.$input-focus-outline-thickness solid post.$input-focus-color;
+  position: relative;
+  z-index: 1;
+}
+
+.star-container {
+  position: relative;
+  display: block;
+  width: post.$lead-font-size;
+  height: post.$lead-font-size;
+  padding: 0 post.$size-micro 0 post.$size-micro;
+  cursor: pointer;
+
+  &.active-star {
+    .fill {
+      color: post.$yellow;
+    }
+    .border {
+      color: post.$black;
+    }
+  }
+
+  &.hovered-star {
+    .fill {
+      color: post.$yellow;
+    }
+    .border {
+      color: post.$gray-80;
+    }
+  }
+
+  &.was-active-star {
+    .fill {
+      color: post.$gray-20;
+    }
+    .border {
+      color: post.$gray-60;
+    }
+  }
+
+  &.default-disabled {
+    cursor: default;
+
+    .fill {
+      color: post.$white;
+    }
+    .border {
+      color: post.$gray-40;
+    }
+  }
+
+  &.active-disabled {
+    cursor: default;
+
+    .fill {
+      color: post.$gray-10;
+    }
+    .border {
+      color: post.$gray-40;
+    }
+  }
+}
+
+.fill {
+  position: absolute;
+  color: post.$white;
+}
+
+.border {
+  width: post.$lead-font-size + 0.125em;
+  height: post.$lead-font-size + 0.125em;
+  position: absolute;
+  top: -0.0625rem;
+  left: post.$size-micro - 0.0625rem;
```

Comment:
Because you ussed `em` for `width` and `height`, you should also use `em` here, instead of `rem`, to make sure, it will always offset dependant on the current `font-size`.